### ISSUE TITLE
Collection medias method returns empty list

### ISSUE THREAD

**Issue by leikoilja** [adw0rd/instagrapi#250]
@adw0rd once again, thanks a lot for your work! 💥 💯 

I think there might be a bug when using `collection_medias` when i want to get all saved objects. According to the documentation it says to specify `amount=0` to get all medias from saved collection. But when i do - an empty list is returned back.
I do
```python
collection_pk = client.collection_pk_by_name(
  collection_name
)
collection_media = client.collection_medias(collection_pk=collection_pk, amount=0)
```
and get `collection_media = []`

I quickly looked at the implementation of `collection_medias(...)` there are lines:
```python
total_items = []
next_max_id = ""
while True:
  if len(total_items) >= float(amount):
  return total_items[:amount]
```
So when `amount=0` and in first iteration `total_items = []` it looks like `if` statement evaluates True and returns an empty list right away. Is it not? 🤔 

As a workaround settings `amount = 999` allows me to download all media at once :)

**Comment by adw0rd** [comment]
@leikoilja thanks for report! I'll try to fix it tomorrow

**Comment by adw0rd** [comment]
Fixed in instagrapi 1.9.12
